app/components/forms/events/view/create-discount-code.js

```diff
@@ -0,0 +1,122 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import $ from 'jquery';
```

Comment:
Don't import it from `jquery`. It's already available as a part of ember at `Ember.$` use that